src/main/java/ca/bc/gov/backendstartapi/endpoint/VegetationCodeEndpoint.java

```diff
@@ -23,6 +31,7 @@
 @RestController
 @RequestMapping(path = "/api/vegetation-codes")
 @Validated
+@Tag(name = "vegetationCode", description = "Codes describing various vegetation species")
 public class VegetationCodeEndpoint {
 
   @Autowired private VegetationCodeRepository vegetationCodeRepository;
```

Comment:
though not related to the swagger documentation changes, more in general, please avoid field injection.